tests/app/questionnaire/routing/test_when_rule_evaluator.py

```diff
@@ -247,6 +249,31 @@ def test_metadata_source(metadata_value, expected_result):
     )
 
 
+@pytest.mark.parametrize(
+    "response_metadata_value, expected_result",
+    [
+        ("2021-10-11T09:40:11.220038+00:00", True),
+        ("2021-10-12T09:40:11.220038+00:00", False),
+    ],
+)
+def test_response_metadata_source(response_metadata_value, expected_result):
+    when_rule_evaluator = get_when_rule_evaluator(
+        response_metadata={"started_at": response_metadata_value},
+    )
+
+    assert (
+        when_rule_evaluator.evaluate(
+            rule={
+                Operator.EQUAL: [
+                    {"source": "response_metadata", "identifier": "started_at"},
+                    "2021-10-11T09:40:11.220038+00:00",
+                ]
+            },
```

Comment:
Normally wouldn't use an operator like this for response metadata, but given we don't support ISO format dates in operators or placeholder yet, I think it is fine for now.